misc/userscripts/qutebrowser_viewsource

```diff
@@ -24,9 +24,15 @@
 # Caveat: Does not use authentication of any kind. Add it in if you want it to.
 #
 
+TERMINAL=${TERMINAL:-urxvt}
+EDITOR=${EDITOR:-vim}
+
 path=/tmp/qutebrowser_$(mktemp XXXXXXXX).html
+curl "$QUTE_URL" > "$path"
 
-curl "$QUTE_URL" > $path
-urxvt -e vim "$path"
+unset _command
+[[ $EDITOR =~ emacs|gedit|leafpad|atom ]] && _command="$EDITOR $path"
+[ -z "$_command" ] && _command="$TERMINAL -e \"vim $path\""
 
+eval "$_command"
```

Comment:
@blyxxyz the reason for doing it funky is for termite. Most terminal emulators take multiple arguments after the `-e` flag, termite only allows one. 

eg, if I had a file readme.txt in my home dir and that was the working dir of a launched terminal emulator:

`st -e vim readme.txt` works
`termite -e vim readme.txt` doesn't work.
`termite -e "vim \"readme.txt\""` would work, however. 

good suggestion on if statement, will update with that.